src/main.rs

```diff
@@ -79,6 +80,10 @@ fn main() {
             .possible_values(&["0", "1"])
             .help("Calculate embeddings in memory or with memory-mapped files")
             .takes_value(true))
+        .arg(Arg::with_name("output-format")
+            .short("f")
```

Comment:
Please add possible and default values, something like:
```
.possible_values(&["textfile", "numpy"])
.default_value("textfile")
```
and remove taking default from `match matches.value_of("output-format").unwrap_or("textfile")`